### ISSUE TITLE
Windows build: init in components not being called

### ISSUE THREAD

**Issue by bojjenclon** [underscorediscovery/luxe#214]
When I try to build my project for windows (using Sublime Text 3), the code compiles fine, but when it runs none of my components seem to be inited properly. It works fine when I build for web; everything gets setup as it should. It could be a misunderstanding on my part of how init is supposed to work, but considering the fact that the behavior is differing between a web build and windows build, this should probably be looked at regardless.

**Comment by underscorediscovery** [comment]
It should be noted that [this](https://github.com/underscorediscovery/luxe/commit/aba7ec99995ba50c33ea925bb0b5df1c83811b16) and [this](https://github.com/underscorediscovery/luxe/commit/c4f57ccacc80b39849ec6e096f2ddaeeeb8e65da) might resolve what you're seeing, but still interested if this is an issue or not.

**Comment by bojjenclon** [comment]
Commenting on this a million years later: I've recently decided to give Luxe another try (as I was too busy with college to keep working with it before), and after updating, everything seems to be working fine. That being said, I moved a few lines around in my code, although nothing major, so I'm not sure if that contributed to this or not. Either way, everything appears to be working correctly now. Sorry about the wait.
